### PR TITLE
Add FAIL, SUCCESS opcode

### LINKED ISSUE #283: Add FAIL opcode
Specification
- opcode: Fail (0x??)
- behavior: The script fails.

### PR DESCRIPTION
**Pull request by junbeomlee**
Fixed #283

### CODE REVIEW

**Comment by kseo** [comment]
@junbeomlee You need to sign the CLA. https://www.clahub.com/agreements/CodeChain-io/codechain

File: vm/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ authors = ["CodeChain Team <codechain@kodebox.io>"]
 
 [dependencies]
 codechain-crypto = { path = "../crypto" }
-codechain-key = { path = "../key" }
+codechain-key = { path = "../key"}
```
Comment (Kais-DkM) [inline vm/Cargo.toml]: It doesn't need to be changed.

**Comment by junbeomlee** [comment]
@kseo Thank you, i signed it!!
@Kais-DkM  I've updated the commit so please check.

File: vm/src/executor.rs [hunk 2]
```diff
@@ -36,6 +36,7 @@ impl Default for Config {
 
 #[derive(Debug, PartialEq)]
 pub enum ScriptResult {
+    Success,
```
Comment (Kais-DkM) [inline vm/src/executor.rs]: We don't need additional result type. `Unlocked` means `Success`.

File: vm/src/executor.rs [hunk 3]
```diff
@@ -161,6 +162,8 @@ pub fn execute(
         match &script[pc] {
             Instruction::Nop => {}
             Instruction::Burn => return Ok(ScriptResult::Burnt),
+            Instruction::Success => return Ok(ScriptResult::Success),
```
Comment (Kais-DkM) [inline vm/src/executor.rs]: Return `Unlocked` instead of `Success`.

**Pull request merged**